tools/cldr-code/src/main/java/org/unicode/cldr/util/BNF.java

```diff
@@ -82,8 +81,7 @@ public BNF complete() {
             if (msg.length() != 0) msg = "Error: Missing definitions for: " + msg;
             String temp = showDiff(ruleSet, variables);
             if (temp.length() != 0) temp = "Warning: Defined but not used: " + temp;
```

Comment:
This is the kind of code inconsistency that bugs me in the current code base. With a mix of braced and un-braced blocks it would be so easy to end up with misleading code.

If/when we decide on a formatter and are willing to reformat everything, cleaning some of this stuff up becomes much easier:
1: adopt a style
2: run a tool to apply it everywhere
3: reformat with the standard formatter

This decouples tools which structurally modify code from "simple" formatting (which if we trust the formatter) we can just unconditionally run as a final step in any refactoring. Otherwise we might end up deciding to not use a refactoring tool because we don't like how it formats its output.

Reply:
This is odd. I think there is an eclipse setting we weere using to add missing brackets, so I'm surprised to see this.